lib/commands/debug.ts

```diff
@@ -21,6 +21,8 @@
 				return (() => {
 					let projectData: IProjectData = this.$injector.resolve("projectData");
 
+					this.$logger.info(`Successfully synced application ${deviceAppData.appIdentifier} on device ${deviceAppData.device.deviceInfo.identifier}.`);
```

Comment:
Good suggestion. I had some issues doing this in the current implementation of the livesync service. That is why I will do this in another commit.